packages/vaadin-lumo-styles/gulpfile.js

```diff
@@ -159,7 +159,7 @@ Iconset.register('lumo', 1000, template);\n`;
   font-style: normal;
 }
 
-html {
+:where(:root, :host) {
```

Comment:
This should only apply to CSS file or JS also? Maybe we should explicitly keep old syntax for JS for now.

Reply:
Done.